.github/workflows/basemap-for-windows.yml

```diff
@@ -164,10 +164,11 @@ jobs:
         name: Build wheel
         run: |
           cd ${{ env.PKGDIR }}
-          $env:GEOS_DIR = "extern"
-          $env:NUMPY_INCLUDE_PATH = "extern/include"
+          $env:GEOS_DIR = "$env:GITHUB_WORKSPACE/${{ env.PKGDIR }}/extern"
+          $env:NUMPY_INCLUDE_PATH = "$env:GITHUB_WORKSPACE/${{ env.PKGDIR }}/extern/include"
           pip install -r requirements-setup.txt
-          python setup.py sdist bdist_wheel
+          python setup.py sdist
+          pip wheel -w dist --no-deps (Get-Item ${{ env.PKGDIR }}/dist/*.zip)
```

Comment:
Wouldn't the `cd ${{ env.PKKGDIR }}` on line 166 make the `${{ env.PKGDIR }}` in the `Get-Item` on line 171 unnecessary?

Reply:
Yes, you are totally right. I messed it up there.